### PR TITLE
README refactor

### PR DESCRIPTION
**Pull request by kethinov**
@mulligan121 can you review this to ensure it is accurate?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,96 @@
...
+logger.log('some info')
+logger.warn('a warning')
+logger.error('an error')
+logger.verbose('noisy log only displayed when verbose is enabled')
+logger.dbError('custom log for database errors')
 ```
-npm run coverage
-```
+
+[TODO: show output of above example.]
```
Comment (mulligan121) [inline README.md]:
```
$ some info
$ ⚠️  a warning
$ ❌  an error
$ noisy log only displayed when verbose is enabled
$ ❌  custom log for database errors
```

File: README.md [hunk 2]
```diff
@@ -1,74 +1,96 @@
...
+
+- `warnings` *[Boolean]*: Enable logging of warnings.
+
+  - Default: `true`.
+
+- `verbose` *[Boolean]*: Enable verbose (noisy) logging.
+
+  - Default: `false`.
+
+- `disable` *[Array of Strings]*: Disable logging in certain environments.
```
Comment (lannonbr) [inline README.md]: Can it be clarified further what the strings in this array refer to? I see the example is an environment variable but can other strings be set as well?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kethinov) [reply README.md]: go for it

File: README.md (hunk 2 above)
Comment (kethinov) [reply README.md]: @mulligan121 can you address this question

**Comment by lannonbr** [comment]
Closing due to changes added in #11

**Pull request closed without merging**